src/stories/3-fields/wrapper/wrapper.stories.tsx

```diff
@@ -28,6 +28,7 @@ export default {
 	},
 	argTypes: {
 		...ExcludeReactFormHookProps,
+		...CommonFieldStoryProps("wrapper"),
```

Comment:
`CommonFieldStoryProps` will add the `label` prop which is not present in `wrapper`, so there is a need to disable that prop in this story